apps/ehr/test/e2e-utils/resource-handler.ts

```diff
@@ -35,6 +36,7 @@ export const PATIENT_CITY = 'New York';
 export const PATIENT_LINE = '10 Cooper Square';
 export const PATIENT_STATE = 'NY';
 export const PATIENT_POSTALCODE = '06001';
+export const INSURANCE_NAME = 'Test insurance' + randomUUID();
```

Comment:
Random strings cause problems often (because they are different in different threads). We could use today's date in format YYYY-MM-DD probably. Does that work for you?